main.go

```diff
@@ -184,6 +192,10 @@ func CreateExecutionPlan(cfg *Config) ([]string, error) {
 			args = append(args, "--memory", cfg.Memory)
 		}
 
+		if cfg.Timeout != "" {
+			args = append(args, "--timeout", cfg.Timeout)
+		}
+
```

Comment:
We noticed this was missing. Please let us know if this is ok.

Reply:
👍 

Can you add it to the example in the README plz?